projects/ccd-case-ui-toolkit/src/lib/shared/services/case-flag/case-flag-refdata.service.ts

```diff
@@ -35,6 +36,12 @@ export class CaseFlagRefdataService {
         welshRequired ? url += 'welsh-required=Y' : url += 'welsh-required=N';
       }
 
+      if (externalFlagsOnly) {
+        // Check if flag-type has been added to the query string; if so, append welsh-required with '&'
+        url.indexOf('?') > -1 ? url += '&' : url += '?';
+        url += 'external-flags-only=true';
```

Comment:
@MateiHMCTS - just to double-check: is it definitely `external-flags-only=true` rather than `external-flags-only=Y`?

Reply:
Checked; seems fine


ExternalFlagsOnly | A boolean to indicate that only flags with the attribute AvailableExternally set to true in the FlagServices table should be returned